openmdao/devtools/iprof_mem.py

```diff
@@ -39,25 +39,17 @@ def _trace_mem_ret(frame, arg, stack, context):
         # print("%g (+%g) MB %s:%d:%s" % (usage, delta,
         #                                 key[0], key[1], key[2]))
 
-def setup(methods=None):
-    """
-    Setup memory profiling.
-
-    Parameters
-    ----------
-    methods : list of (glob, (classes...)) or None
-        Methods to be profiled, based on glob patterns and isinstance checks.
-    """
+def _setup(options):
```

Comment:
I guess we don't require docstrings on private or dev-specific stuff, but it seems like there is a small audience which might fight them helpful...

Reply:
I am part of this small audience.